static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -173,6 +181,11 @@
     background: $paid-article-subheader;
     border-top-color: $paid-article-brand;
 
+    &:hover,
+    &:focus {
+        background: darken($paid-article-subheader, 7%);
```

Comment:
what I mean is to have some rules, that darken can be 5% or 10% only. So we won't end up with gazillion shades of grey. It is really a minor thing but it will add up and guardian website shouldn't be a circus but some solid website with patterns.